### DISCUSSION TITLE
AIR 32

### DISCUSSION THREAD

**Discussion by hiPeeps124816** [airsdk/Adobe-Runtime-Support#872] [Q&A]
How do I install AIR 32?

**Comment by ajwfrost** [comment]
https://helpx.adobe.com/uk/air/kb/archived-air-sdk-version.html

I'm not sure how much longer this link will stay active for though..

**Reply by hiPeeps124816** [reply]
But how do I open it when it's just a folder?

**Reply by ajwfrost** [reply]
What are you trying to install? the SDK, or the runtime? and on which platform?
The SDKs are provided as a zip for Windows, or dmg file for macOS; you just have to extract the zip somewhere to use it on Windows, and copy the folder out of the dmg file into somewhere useful on macOS.
If it's for the runtime then these should be installation applications, the macOS one may not work on Catalina/Big Sur though..

**Reply by hiPeeps124816** [reply]
I'm trying to install the sdk on macOS 10.15.7 because I can't find the air 32 runtime. When I open the dmg, it turns into a folder.

**Reply by ajwfrost** [reply]
Yes, that would be how the SDK is deployed, it's just a folder that's provided within the DMG file.. 
Do you have a .air file? as you can perhaps use the SDK to turn the .air file into an application bundle. With macOS 10.15 I think you would need to copy the contents from the DMG file onto your filesystem somewhere and then go into a terminal window, find this new location and do:
```
xattr -r -d com.apple.quarantine .
```
to remove the quarantine flag that you get when downloading files from the internet..

Then you can use the bin/adt tool to install or modify your .air file ..

**Reply by hiPeeps124816** [reply]
I just need it to run the Scratch 2 offline editor because AIR 33 gives it a white screen.

**Reply by ajwfrost** [reply]
Hi - I just downloaded and installed the latest AIR 33 runtime on a Big Sur machine, from https://airsdk.harman.com/assets/downloads/AdobeAIR.dmg, and once I'd installed that I opened the dmg file that I get from the Scratch website for the Scratch 2 offline editor: it then allows me to install - but not run - their application.
Then going into a terminal window to /Applications, I ran:
` sudo xattr -r -d com.apple.quarantine Scratch\ 2.app`
And then, running it from Finder worked fine....
